bugbug/model.py

```diff
@@ -367,6 +367,15 @@ def items_gen(self, classes):
 
             assert len(commit_map) > 0
 
+        if "to_rollback" in classes:
+            for bug in bugzilla.get_bugs():
+                bug_id = bug["id"]
+
+                if bug_id not in classes["to_rollback"]:
+                    continue
+
```

Comment:
@marco-c 
1.one solution could be to use bug_snapshot.rollback() here , and not use it in pipeline.This way it won't matter what bugs yield first,
2. We can introduce a new field to the bugs we want to rollback(cause we need something to differentiate them from the original bug) , and rollback them in bug_features.py, but we might want to remove "already_rollbacked" parameter this way.